### PR TITLE
feat: capture invoice activity

### PR DESCRIPTION
**Pull request by Dev-Nino**
Ticket ID: [#141](https://github.com/nerubia/kh-books/issues/141)

### CODE REVIEW

File: src/controllers/khbooks/invoice-activity-controller.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/* eslint-disable unused-imports/no-unused-vars */
```
Comment (jatnerubia) [inline src/controllers/khbooks/invoice-activity-controller.ts:1]: let's remove this ^^

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a minor change ^^

File: src/controllers/khbooks/invoice-activity-controller.ts (hunk 1 above)
Comment (Dev-Nino) [reply src/controllers/khbooks/invoice-activity-controller.ts]: done removing J ⤴️

**Review by jatnerubia — APPROVED** [review]
Cool

**Pull request merged**